Source/WTF/wtf/RandomNumber.cpp

```diff
@@ -41,6 +40,17 @@ double randomNumber()
     return static_cast<double>(bits) / (static_cast<double>(std::numeric_limits<uint32_t>::max()) + 1.0);
 }
 
+unsigned randomUint32()
+{
+    return cryptographicallyRandomNumber();
+}
+
+uint64_t randomUint64()
+{
+    auto high = static_cast<uint64_t>(cryptographicallyRandomNumber());
+    return (high << 32) | cryptographicallyRandomNumber();
+}
```

Comment:
Let's name it `cryptographicallyRandomUint64` etc. to emphasis it is cryptographically random.
We should not use it when cryptographically random numbers are required (if weak-random is enough, then use it).

Reply:
The convention in this file is "random" means cryptographically random while "weakRandom" means weak random. If I hadn't looked at the implementation I wouldn't have known that as the documentation doesn't imply that.

My opinion would be to keep the naming and then do an audit of where `random*` is used to figure out if they should be cryptographically random or not.

Reply:
However we have real problem that, since `randomNumber` name looks so canonical and looks easy, many places were wrongly using that function. And the problem was that it is surprisingly costly function.
When working on Speedometer2.1 improvement, profiler trace showed many strong random generations, and it turned out most of them are OK with weak random. Not to repeat this mistake, we should name strong version of this well with the descriptive name to require code contributors to think about which one should be picked in that callsite.

Reply:
@donny-dont I would name the new function as @Constellation suggests, even though it does not parallel the naming of the existing function, then report a follow-up bug to rename the existing function.